store/postgres/src/store.rs

```diff
@@ -283,9 +283,7 @@ impl BasicStore for Store {
         // Add specified filter to query
         if let Some(filter) = query.filter {
             diesel_query = store_filter(diesel_query, filter).map_err(|e| {
-                error!(self.logger, "value does not support this filter";
-                                    "value" => format!("{:?}", e.value),
-                                    "filter" => e.filter)
+                QueryExecutionError::FilterNotSupportedError(format!("{:?}", e.value), e.filter)
```

Comment:
We're not going to send debug-formatted values to clients. We have to come up with something better.